ynr/apps/people/tests/test_person_update.py

```diff
@@ -547,3 +546,16 @@ def test_biography_last_updated_timestamp(self):
             response,
             "This statement was last updated on {}.".format(last_updated),
         )
+        # make a second edit to the same person and ensure
+        # the biography timestamp from the previous edit is still shown
+        form["name"] = "Bess Palmer"
+        form["source"] = "Mumsnet"
+        form.submit()
+        self.person.refresh_from_db()
+
+        self.assertEqual(len(self.person.versions), 2)
+        self.assertEqual(self.person.name, "Bess Palmer")
+        self.assertNotEqual(self.person.biography_last_updated, None)
+        response = self.app.get(
+            "/person/{}/update".format(self.person.pk), user=self.user
+        )
```

Comment:
This is currently redundant. to the test. However I would use it to repeat the fragment checking `self.person.biography_last_updated` on lines 542-548, to make sure the time *hasn't* been updated.

Reply:
Typo on both comments, thank you. I have not cleaned up as suggested.